tests/functional_test.js

```diff
@@ -1,41 +1,41 @@
 'use strict';
 
-var path   = require('path');
+var path = require('path');
 var assert = require('assert');
-var walk   = require('fs-walk');
-var async  = require('async');
+var walk = require('fs-walk');
+var async = require('async');
 
-var digest  = require(path.join(__dirname, '..', 'lib', 'helpers')).sri.digest;
+var digest = require(path.join(__dirname, '..', 'lib', 'helpers')).sri.digest;
 var helpers = require(path.join(__dirname, 'test_helper'));
-var config  = helpers.config();
+var config = helpers.config();
 
 var expectedHeaders = {
-    date: undefined,
-    etag: undefined,
-    expires: undefined,
+    'date': undefined,
```

Comment:
```
"quote-props": [2, "consistent"]
```

Feel free to check out and suggest any other options. I'm not tied to these ones, just tried to be as strict as possible and close to my own personal style.